src/main/java/io/cdap/plugin/gcp/publisher/source/GoogleSubscriber.java

```diff
@@ -110,22 +126,80 @@ public void prepareRun(StreamingSourceContext context) throws Exception {
     ClassTag<SparkPubsubMessage> tag = scala.reflect.ClassTag$.MODULE$.apply(SparkPubsubMessage.class);
     JavaReceiverInputDStream<SparkPubsubMessage> pubSubMessages = new JavaReceiverInputDStream<>(stream, tag);
 
+    Schema customMessageSchema = getCustomMessageSchema();
+    final Schema outputSchema = config.getSchema();
+    final String format = config.format.toLowerCase();
```

Comment:
use getFormat() instead of accessing format directly.

Reply:
PR updated.